access/src/main/java/zone/cogni/asquare/virtuoso/VirtuosoRdfStoreService.java

```diff
@@ -51,6 +36,10 @@ public class VirtuosoRdfStoreService implements RdfStoreService {
   private final boolean graphCrudUseBasicAuth;
   protected String modelContentType;
 
+  private HttpClient httpClient;
+
+  private QueryExecutionBuilder queryExecutionBuilder;
```

Comment:
is this builder threadsafe? assuming we do not update its properties.

Reply:
Good catch - checked the implementation and `QueryExecutionBuilderAdapter` mutates its content, so extracted it to the builder methods.